m2cgen/assemblers/linear.py

```diff
@@ -33,7 +33,8 @@ def _get_coef(self):
 class SklearnLinearModelAssembler(BaseLinearModelAssembler):
 
     def _get_intercept(self):
-        return self.model.intercept_
+        return getattr(self.model, "intercept_",
```

Comment:
Is it worth to have a `LightingLinearModelAssembler` version of this assembler which inherits `SklearnLinearModelAssembler` and overrides the `_get_intercept` method with:
```
def _get_intercept(self):
    return np.zeros(self._get_coef().shape[0])
```
This way our expectations to those libraries will be clearly defined.

Reply:
It will override this method in the following way:
```
    def _get_intercept(self):
        return getattr(self.model, "intercept_",
                       np.zeros(self._get_coef().shape[0]))
```
In `lightning`, model can have `intercept_` attribute and can have no this attribute. In `scikit-learn`, model has `intercept_` attribute in all cases. Please tell me whether you still want to introduce `LightingLinearModelAssembler` or not after the info above.

Reply:
Ah, I see. Thanks for providing this context. Let's keep it as is then.